docs/_articles/en/getting-started/install.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Salesforce Extensions for VS Code
+title: Installation and Java Setup
```

Comment:
@ekapner - I sort of liked the old title better.  I think most people use the search bar.  (not sure if you were thinking we should mention 'java' for discoverability?)